### DISCUSSION TITLE
Login storefront /admin  with Facebook, Google account

### DISCUSSION THREAD

**Discussion by nhatanhit** [medusajs/medusa#4310] [Feature Requests]
Hi Medusa team. 
We have the requirement: A user / customer  can login with Facebook / Google Account into Medusa admin / storefront site. We don't know if those features are available in Medusa or not. 
If they are available , can you please provide the guideline how to setup them on Medusa system. 
If they are not available for now, do you have the plan to create those features ? , or can you please suggest some approach to implement them at the moment 
Thank you

**Comment by Max-access89** [comment]
Hi, was this ever resolved?
